base/shell/explorer/traywnd.cpp

```diff
@@ -264,6 +264,115 @@ class CStartButton
 
 };
 
+static const WCHAR szTrayShowDesktopButton[] = L"TrayShowDesktopButtonWClass";
```

Comment:
Is it an invented name? Or is this the class name of the corresponding button on Windows 7+?

Reply:
It's confirmed by WinHier on Win10.

![image](https://user-images.githubusercontent.com/2107452/191405488-6144459f-3e76-4cb6-8b76-b085aaef77e0.png)

Reply:
Ah, and yes, as shown in your screenshot (which, by the way, gives the same window hierarchy as on Windows 7), we can see that the show-desktop button is *within* the trayNotifyWnd, i.e. next to the clock etc. within that tray-window, as shown in my screenshots elsewhere in this PR. Not outside as seems to be from your screenshots in the PR description.

Reply:
Should we touch `TrayNotifyWnd`? It's a complicated hierarchy.